docs/vmalert.md

```diff
@@ -89,20 +102,21 @@ groups:
 ### Groups
 
 Each group has the following attributes:
+
 ```yaml
 # The name of the group. Must be unique within a file.
 name: <string>
 
-# How often rules in the group are evaluated.
-[ interval: <duration> | default = -evaluationInterval flag ]
+  # How often rules in the group are evaluated.
```

Comment:
looks like some auto-formatter broke the indentations in the README file. Would you mind to revert it and apply changes only to flags section?

Reply:
sorry something went wrong with formatting, fixed